### PR TITLE
AND-7316: Show the number of collaborators in avatarView correctly

### PR DESCRIPTION
**Pull request by nyjalusc**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @nyjalusc has signed the CLA. Thanks for the pull request!

File: box-content-sdk/src/main/java/com/box/androidsdk/content/views/BoxAvatarView.java [hunk 1]
```diff
@@ -117,7 +115,17 @@ public void run() {
             } else if (SdkUtils.isBlank(name) && mUser instanceof BoxUser){
                 name = ((BoxUser) mUser).getLogin();
             }
-            SdkUtils.setInitialsThumb(getContext(), mInitials, name);
+            int numberOfCollab = 0;
+            try {
+                numberOfCollab = Integer.parseInt(name);
+            } catch (NumberFormatException ex) {
```
Comment (rgcorrea) [inline box-content-sdk/src/main/java/com/box/androidsdk/content/views/BoxAvatarView.java]: Can this "name" variable really contain a number ?

Comment (nyjalusc) [inline box-content-sdk/src/main/java/com/box/androidsdk/content/views/BoxAvatarView.java]: Yes

File: box-content-sdk/src/main/res/drawable-hdpi/boxsdk_thumb_background.xml [hunk 2]
```diff
@@ -1,4 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <shape xmlns:android="http://schemas.android.com/apk/res/android" android:shape="oval" >
     <solid android:color="#FFFFFFFF" />
+    <stroke
+        android:width="3px"
```
Comment (doncung) [inline box-content-sdk/src/main/res/drawable-hdpi/boxsdk_thumb_background.xml]: Can you use dp? Generally we should not define things with px.

Comment (nyjalusc) [inline box-content-sdk/src/main/res/drawable-hdpi/boxsdk_thumb_background.xml]: +1

**Comment by doncung** [comment]
+1

**Pull request merged**